include/circt/Dialect/FIRRTL/FIRRTLUtils.h

```diff
@@ -322,6 +322,20 @@ static ResultTy transformReduce(MLIRContext *context, RangeTy &&r,
 /// Truncate `a` to the common prefix of `a` and `b`.
 void makeCommonPrefix(SmallString<64> &a, StringRef b);
 
+//===----------------------------------------------------------------------===//
+// OM dialect utilities
+//===----------------------------------------------------------------------===//
+
+/// Create a ClassOp, with the specified fieldNames and fieldTypes as ports. The
+/// output property is set from the input property port.
+ClassOp buildSimpleClassOp(OpBuilder &odsBuilder, Location loc, Twine name,
```

Comment:
Could you define this as a class op builder method?

Reply:
Thanks for the suggestion, moved it to a class op builder.